core/src/main/scala/io/finch/request/package.scala

```diff
@@ -162,6 +180,8 @@ package object request {
       def apply[Req](req: Req)(implicit ev: Req => HttpRequest) =
         try number.toFuture
         catch { case _: IllegalArgumentException => ValidationFailed(param, rule).toFutureException }
+
+      override def toString: String = s"param: $param"
```

Comment:
I can reopen it to be merged. If we want to do that we should decide between `should` and `which`. I prefer `should`.